x-pack/plugin/mapper-aggregate-metric/src/test/java/org/elasticsearch/xpack/aggregatemetric/mapper/AggregateDoubleMetricFieldMapperTests.java

```diff
@@ -506,6 +572,9 @@ private Map<String, Object> randomAggregateMetric() {
         private void mapping(XContentBuilder b) throws IOException {
             String[] metrics = storedMetrics.stream().map(Metric::toString).toArray(String[]::new);
             b.field("type", CONTENT_TYPE).array(METRICS_FIELD, metrics).field(DEFAULT_METRIC, metrics[0]);
+            if (malformedExample) {
+                b.field(IGNORE_MALFORMED, true);
+            }
         }
 
```

Comment:
Can we add some more elaborate use cases, here or in yaml? We want to check for:
 - Nested subfields within aggregate_metric_double
 - Partial aggregate_metric_double (e.g. one metric ok, one invalid, two ok)
 - Nested malformed fields, with other fields at the same level

Reply:
We do cover partials in `malformedValue`. Nested is a good suggestion.